digitiser-aggregator/src/main.rs

```diff
@@ -56,6 +60,7 @@ struct Cli {
     #[clap(long, default_value = "500")]
     cache_poll_ms: u64,
 
+    /// Kafka store for metrics
```

Comment:
I don't get this? This is the endpoint on which OpenMetrics flavour metrics are available. This has nothing to do with Kafka.